base/cvd/cuttlefish/host/commands/assemble_cvd/flags/mcu_config_path.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <json/value.h>
+
+#include "cuttlefish/common/libs/utils/result.h"
+
+namespace cuttlefish {
+
+class McuConfigPathFlag {
+ public:
+  static McuConfigPathFlag FromGlobalGflags();
+
+  Result<Json::Value> JsonForIndex(size_t) const;
```

Comment:
minor nit: i think you want `#include <cstddef>` for `size_t`

Reply:
Good catch, thanks.

Since the [style guide](https://google.github.io/styleguide/cppguide.html#Integer_Types) mentions using the types without the `std::` prefix, used `<stddef.h>` rather than `<csstddef>`.